lib/formatters.rb

```diff
@@ -186,7 +186,22 @@ def header(data, fasta_mapper = nil)
       CSV.generate do |csv|
         first = data.first
         keys = fasta_mapper ? ['fasta_header'] : []
-        csv << (keys + first.keys).map(&:to_s) if first
+
+        firstKeys = first.keys
```

Comment:
Can't we reuse the `keys` variable for this? Something like `keys += first.keys`?

_[Original comment](https://github.ugent.be/unipept/unipept-cli/pull/55#discussion_r21550) by @bmesuere on Thu May 23 2019 at 10:46._